src/EditorFeatures/Test2/InlineParameterNameHints/CSharpInlineParamNameHintsTests.vb

```diff
@@ -0,0 +1,272 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.InlineParameterNameHints
+    Public Class CSharpInlineParamNameHintsTests
+        Inherits AbstractInlineParamNameHintsTests
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestNoParameterSimpleCase() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+class A
+{
+    int testMethod()
+    {
+        return 5;
+    }
+    void Main() 
+    {
+        $$testMethod();
```

Comment:
We can remove all of the $$s in the file right, because we're not using cursor location any more?

Reply:
They've all been removed.